index.js

```diff
@@ -6,12 +6,81 @@
  */
  export function fhirconvert(str, vars) {
   if (validate(str, vars)) {
-    return varfind(convert(str.trim()), vars);
+    return varfind(convert(str), vars);
   } else {
     return null;
   }
 }
 
+// Array of usable operators
+let ops = [
```

Comment:
Looks like this and others below could be const, unless they get set outside this file, no?

Reply:
I do see there's a mix of var and let